sw/airborne/boards/apogee/chibios/v1.0/board.cfg

```diff
@@ -188,146 +188,185 @@ HEADER
 /*
  * PWM defines
  */
+#if defined(LINE_SERVO0)
```

Comment:
Only for the Apogee that is using a single h file, since the header from CFG is placed before ChibiOS defines, you can't add these lines otherwise PWM outputs are not defined. You shouldn't loose time to properly split the files for this board, so just remove the `#if defined(LINE_SERVOX)` protection and it should be fine.
It seems to work correctly for the Tawaki/F7 board.

Reply:
I think the apogee board was just cursed to fail every commit. But I corrected the mistake and it should work now